Package.cs

```diff
@@ -3,5 +3,6 @@ public enum Package {
     Main,
     Gehenna,
     Prototype,
+    All

```

Comment:
I don't think this is necessary at the moment. Let's leave it out until you need it.